docs/products/postgresql/concepts/high-availability.md

```diff
@@ -51,26 +55,27 @@ primary node.
 
 ## Highly available Business and Premium service plans
 
-When the failed node is a PostgreSQL **standby** node, the primary node
+When the failed node is a PostgreSQL standby node, the primary node
 keeps running normally and provides a normal service level to the client
 applications. Once the new replacement standby node is ready and
 synchronised with the primary node, it starts replicating the primary
 node in real time as the situation reverts back to normal.
 
-When the failed node is a PostgreSQL **primary** node, the combined
+When the failed node is a PostgreSQL primary node, the combined
 information from the Aiven monitoring infrastructure and the standby
 node is used to make a failover decision. The standby node is then
 promoted as the new primary and immediately starts serving clients. A
 new replacement node is automatically scheduled and becomes the new
-standby node.
+standby node. There is no data loss in this scenario.
 
-If all the **primary** and **standby nodes** fail at the same time, new
+If all the primary and standby nodes fail at the same time or if all the primary
```

Comment:
There is only one primary node at any given time. I am not sure I follow this part. I suggest to rever to the previous version.